### PR TITLE
docs: editorial pass on etcd-operator user docs

### PR DESCRIPTION
**Pull request by zbwright**
Just an editorial pass to prepare these docs for publication to the site. Titles may change, but content should be stable.

[skip ci]

### CODE REVIEW

File: doc/user/install_guide.md [hunk 1]
```diff
@@ -1,36 +1,38 @@
-# Operation Guide
+# Getting started guide
```
Comment (hongchaodeng) [inline doc/user/install_guide.md]: Getting started guide -> Install Guide

File: doc/user/client_service.md [hunk 2]
```diff
@@ -8,24 +10,27 @@ example-etcd-cluster          None           <none>        2380/TCP   1m
 example-etcd-cluster-client   10.0.222.115   <none>        2379/TCP   1m
 ```
 
-The client service is of type `ClusterIP` and only accessible from within the Kubernetes cluster's network.
+The client service is of type `ClusterIP` and accessible only from within the Kubernetes cluster's network.
```
Comment (hongchaodeng) [inline doc/user/client_service.md]: Kubernetes cluster's network -> Kubernetes overlay network.

File: doc/user/client_service.md [hunk 3]
```diff
@@ -8,24 +10,27 @@ example-etcd-cluster          None           <none>        2380/TCP   1m
...
+
+For example, access the service from a pod in in the cluster:
```
Comment (hongchaodeng) [inline doc/user/client_service.md]: in ~~in~~

File: doc/user/cluster_tls.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
-# Cluster TLS guide
+# Cluster TLS policy
```
Comment (hongchaodeng) [inline doc/user/cluster_tls.md]: Policy

File: doc/user/cluster_tls.md [hunk 5]
```diff
@@ -56,56 +54,66 @@ The client TLS assets should have the following:
...
 $ kubectl create secret generic etcd-server-tls --from-file=server-ca.crt --from-file=server.crt --from-file=server.key
 ```
 
 etcd-operator will mount this secret at `/etc/etcdtls/member/server-tls/` for each etcd member pod in the cluster.
 
-
 ### operatorSecret
 
-Operator needs to send client requests e.g. snapshot, healthy check, add/remove member in order to maintain this cluster.
+The operator must send the client requests (such as snapshot, healthy check, add/remove member) to maintain this cluster.
```
Comment (hongchaodeng) [inline doc/user/cluster_tls.md]: send ~~the~~ client requests

File: doc/user/client_service.md [hunk 6]
```diff
@@ -1,5 +1,7 @@
-## Client service
-For every etcd cluster created the etcd-operator will create an etcd client service in the same namespace with the name `<cluster-name>-client` .
+# Etcd client service
```
Comment (xiang90) [inline doc/user/client_service.md]: Etcd -> etcd. We always use `etcd` even if it is the first word of the sentence.

File: doc/user/cluster_tls.md (hunk 4 above)
Comment (xiang90) [reply doc/user/cluster_tls.md]: it should be policy, not Policy. We do not capitalize word in headlines.

File: doc/user/cluster_tls.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
 
-Cluster TLS policy is configured on a per-cluster basis via the CR spec provided to etcd-operator.
-For etcd's TLS support and requirements, see [etcd security guide](https://coreos.com/etcd/docs/latest/op-guide/security.html).
-To learn about generating self-signed TLS certs, see [this tutorial](https://coreos.com/os/docs/latest/generate-self-signed-certificates.html).
+Cluster TLS policy is configured on a per-cluster basis through the CR spec provided to etcd-operator.
 
+For etcd's TLS support and requirements, see the [etcd security model][etcd-security].
+To learn about generating self-signed TLS certs, see [Generate self-signed certificates][self-signed].
 
 ## Static cluster TLS Policy
```
Comment (xiang90) [inline doc/user/cluster_tls.md]: Policy -> policy.

File: doc/user/cluster_tls.md [hunk 8]
```diff
@@ -27,27 +27,25 @@ spec:
...
 The peer TLS assets should have the following:
 - **peer.crt**: peer communication cert.
   The certificate should allow wildcard domain `*.${clusterName}.${namespace}.svc` and `*.${clusterName}.${namespace}.svc.${cluster_domain}`.
-  In our case, it is `*.example.default.svc` and `*.example.default.svc.cluster.local`.
+  The example shown above uses `*.example.default.svc` and `*.example.default.svc.cluster.local`.
 - **peer.key**: peer communication key.
 - **peer-ca.crt**: CA cert for this peer key-cert pair.
 
-Create a secret containing those:
+Create a secret using the `peer.key` and `peer-ca.crt`:
```
Comment (hongchaodeng) [inline doc/user/cluster_tls.md]: Actually there is another -- `peer.crt`

File: doc/user/cluster_tls.md [hunk 9]
```diff
@@ -56,56 +54,66 @@ The client TLS assets should have the following:
...
   The certificate should allow wildcard domain `*.${clusterName}.${namespace}.svc`,
   `${clusterName}-client.${namespace}.svc`, and `localhost`.
-  In this case, it is `*.example.default.svc`, `example-client.default.svc`, and `localhost`.
+  The example shown above uses `*.example.default.svc`, `example-client.default.svc`, and `localhost`.
   To use more DNS name or IP to access etcd server, please add it here.
 - **server.key**: etcd server's client communication key.
 - **server-ca.crt**: CA cert for validating the certs of etcd clients.
 
-Create a secret containing those:
+Create a secret using `server.key` and `server-ca.crt`:
```
Comment (hongchaodeng) [inline doc/user/cluster_tls.md]: missing `server.crt`

File: doc/user/cluster_tls.md [hunk 10]
```diff
@@ -56,56 +54,66 @@ The client TLS assets should have the following:
...
 ```
 etcd-client.crt
 etcd-client.key
 etcd-client-ca.crt
 ```
-Both `etcd-client.crt` and `etcd-client.key` should trusted by etcd server's client CA `server-ca.crt`.
-`etcd-client-ca.crt` should trust etcd server's key and cert.
 
-Here is an example `etcdctl` command to list members from the secure etcd cluster:
+Both `etcd-client.crt` and `etcd-client.key` should be trusted by the etcd server's client CA `server-ca.crt`.
```
Comment (hongchaodeng) [inline doc/user/cluster_tls.md]: ~~Both~~

File: doc/user/cluster_tls.md [hunk 11]
```diff
@@ -56,56 +54,66 @@ The client TLS assets should have the following:
...
+
+`etcd-client-ca.crt` should trust the etcd server's key and cert.
```
Comment (hongchaodeng) [inline doc/user/cluster_tls.md]:
```
`server.crt` and `server.key` should be trusted by 
the etcd client CA `etcd-client-ca.crt`.
```

File: doc/user/cluster_tls.md [hunk 12]
```diff
@@ -56,56 +54,66 @@ The client TLS assets should have the following:
...
 $ ETCDCTL_API=3 etcdctl --endpoints=https://example-client.default.svc:2379 \
     --cert=etcd-client.crt --key=etcd-client.key --cacert=etcd-client-ca.crt \
     member list -w table
 ```
-It should be run within a pod inside k8s in order to access the service name.
+
+
+[etcd-security]: https://coreos.com/etcd/docs/latest/op-guide/security.html
+[self-signed]: https://coreos.com/os/docs/latest/generate-self-signed-certificates.html
+[example-tls]: https://github.com/coreos/etcd-operator/tree/master/example/tls/
```
Comment (hongchaodeng) [inline doc/user/cluster_tls.md]: use relative path link:
```
../../example/tls/
```

File: doc/user/install_guide.md [hunk 13]
```diff
@@ -39,10 +41,12 @@ kubectl delete clusterrole etcd-operator
...
 **Disclaimer:** The following Helm chart is an external project not maintained by the etcd-operator maintainers; so it may not be up to date.
 
-etcd-operator is available as a [Helm
-chart](https://github.com/kubernetes/charts/tree/master/stable/etcd-operator).
-Follow the instructions on the chart to install etcd-operator on your cluster.
- 
+etcd-operator is available as a [Helm chart][etcd-helm]. Follow the instructions on the chart to install etcd-operator on clusters.
+
+
+[rbac-rules]: rbac.html
```
Comment (hongchaodeng) [inline doc/user/install_guide.md]: rbac.html -> rbac.md

**Comment by fanminshi** [comment]
@zbwright I think it is best to review each doc individually. a PR for a doc file. In this way, it allows us to focus one thing at a time and do it well.

**Comment by zbwright** [comment]
no problem, @fanminshi and @hongchaodeng  I will make these changes, merge this PR, and open one per doc

File: doc/user/cluster_tls.md (hunk 12 above)
Comment (zbwright) [reply doc/user/cluster_tls.md]: I don't think that will work when these docs are published to the site. I think we have to use the absolute path for these. I will check.

File: doc/user/install_guide.md (hunk 13 above)
Comment (zbwright) [reply doc/user/install_guide.md]: I think you are correct with this one. I'll check to make sure it'll work once published.

**Comment by zbwright** [comment]
Let me know if you would like another PR for further review for these, please.

**Comment by hongchaodeng** [comment]
@zbwright 
Yes. Would like smaller PRs:
- ideally one doc per PR.
- But if changes are small in a doc, feel free to combine more.
- No more than 3 docs per PR unless they are similar changes -- for example, naming changes.

**Comment by fanminshi** [comment]
close this because smaller prs are being submitted.

**Pull request closed without merging**